Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -298,6 +298,11 @@ static bool experimentalFeatureEnabled(const String& key, bool defaultValue = fa
     return cacheDirectoryFileSystemRepresentation("AlternativeServices"_s, ShouldCreateDirectory::No);
 }
 
+String WebsiteDataStore::defaultHSTSDirectory()
```

Comment:
I think we should use `defaultHSTSStorageDirectory` to be aligned with all the other HSTSStorageDirectory names.

Reply:
Fixed.